tests/literals.pyi

```diff
@@ -0,0 +1,6 @@
+from typing import Literal
+
+Literal[None]  # Y061 None inside Literal[...] expression. Replace with None
+Literal[True, None]  # Y061 None inside Literal[...] expression. Replace with Literal[True] | None
+Literal[1, None, "foo"]  # Y061 None inside Literal[...] expression. Replace with Literal[1, 'foo'] | None
+Literal[None, None]  # Y061 None inside Literal[...] expression. Replace with Literal[None] | None
```

Comment:
Ah, the style for all our other error messages is to put code snippets in double quotes, so these should be:

```suggestion
Literal[None]  # Y061 None inside "Literal[...]" expression. Replace with "None"
Literal[True, None]  # Y061 None inside "Literal[...]" expression. Replace with "Literal[True] | None"
Literal[1, None, "foo"]  # Y061 None inside "Literal[...]" expression. Replace with "Literal[1, 'foo'] | None"
Literal[None, None]  # Y061 None inside "Literal[...]" expression. Replace with "Literal[None] | None"
```

Also this last one is... weird. It's also very much an edge case that's unlikely to come up, though, so I suppose I'm okay with it!

Reply:
I wanted to include one test with multiple `None`s - I added one extra `None` to the second to last test and got rid of the last one :)